pages/common/git-sync.md

```diff
@@ -0,0 +1,17 @@
+# git sync
+
+> Sync local branch with remote branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sync>.
+
+- Sync local branch with its remote branch:
+
+`git sync`
+
+- Sync local branch with origin/master:
+
+`git sync origin master`
```

Comment:
```suggestion
`git sync origin main`
```

Reply:
I keep forgetting that the standard name for default branches has been `main` for some time.